tests/opensource-suites/charts/preview/widget-width.test.ts

```diff
@@ -0,0 +1,26 @@
+import {Page} from '@playwright/test';
+
+import datalensTest from '../../../utils/playwright/globalTestDefinition';
+import {openTestPage, slct} from '../../../utils';
+import {ChartKitTableQa, PreviewQa} from '../../../../src/shared';
+import {PreviewUrls} from '../../../constants/test-entities/charts';
+
+datalensTest.describe('Charts Preview', () => {
+    datalensTest.beforeEach(async ({page}: {page: Page}) => {
+        await openTestPage(page, PreviewUrls.FlatTableWithOneColumn);
```

Comment:
It is better to forward urls from config - it will allow you to use the same tests on different installations